lib/commons/color/get-background-color.js

```diff
@@ -207,17 +207,20 @@ color.getRectStack = function(elm) {
 function sortPageBackground(elmStack) {
 	let bodyIndex = elmStack.indexOf(document.body);
 	let bgNodes = elmStack;
+	let sortBodyElement =
+		bodyIndex > 1 || bodyIndex === -1; // only if there are negative z-index elements // only when an element is positioned outside of the body's rect
 
 	if (
-		// Check that the body background is the page's background
-		bodyIndex > 1 && // only if there are negative z-index elements
+		sortBodyElement &&
```

Comment:
Shouldn't this just be `bodyIndex !== 0`, since `bodyIndex` can never be less than -1.

Reply:
`bodyIndex` can be `-1` in some specific scenarios. See https://jsfiddle.net/o3hjdsy8/1/